packages/atomic/src/components/commerce/atomic-commerce-layout/commerce-layout.ts

```diff
@@ -1,5 +1,11 @@
 import {buildSearchLayoutCommon} from '../../common/atomic-layout-section/search-layout';
 
+export const layoutWebComponentTagName = 'atomic-commerce-layout';
+export const containerWebComponentTagName = 'atomic-commerce-interface';
```

Comment:
This makes me realize: now that we have a dedicated `atomic-commerce-recommendation-interface`, we need to make some adjustments to make sure it also supports the `atomic-commerce-layout` (or create a dedicated `atomic-commerce-recommendation-layout` component).